tensorflow_addons/image/tests/color_ops_test.py

```diff
@@ -53,3 +53,24 @@ def test_equalize_channel_first(shape):
     image = tf.ones(shape=shape, dtype=tf.uint8)
     equalized = color_ops.equalize(image, "channels_first")
     np.testing.assert_equal(equalized.numpy(), image.numpy())
+
+
+@pytest.mark.parametrize("dtype", _DTYPES)
+@pytest.mark.parametrize("shape", [(5, 5, 3), (10, 5, 5, 3)])
+def test_sharpness_dtype_shape(dtype, shape):
+    image = np.ones(shape=shape, dtype=dtype)
+    sharp = color_ops.sharpness(tf.constant(image), 0).numpy()
+    np.testing.assert_equal(sharp, image)
+    assert sharp.dtype == image.dtype
+
+
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_sharpness_with_PIL():
```

Comment:
Please add a `@pytest.mark.parametrize("factor", [0, 0.25, 0.5, 0.75, 1])`. This will run the test multiple times.